pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -500,6 +500,7 @@ func (s *ServiceImpl) buildDataConnectionsNavLink(c *contextmodel.ReqContext) *n
 			SubTitle: "Browse and create new connections",
 			Url:      baseUrl + "/add-new-connection",
 			Children: []*navtree.NavLink{},
+			Keywords: []string{"graphite", "loki", "prometheus", "sql", "tempo"},
```

Comment:
@mikkancso not sure exactly which datasources you want to add here? i added the basics, but we can always add more 🤔

Reply:
Based on [this dashboard](https://bi.grafana-ops.net/d/I020tGa4k/data-source-search-term-frequencies?orgId=1&from=now-30d&to=now), I'd also add "json" and "csv".